picard/webservice.py

```diff
@@ -453,7 +453,9 @@ def lookup_discid(self, discid, handler, priority=True, important=True, refresh=
         return self._get_by_id('discid', discid, handler, inc, queryargs={"cdstubs": "no"},
                                priority=priority, important=important, refresh=refresh)
 
-    def _find(self, entitytype, handler, kwargs):
+    def _find(self, entitytype, handler, kwargs,
```

Comment:
I suspect that the issue causing the error may be that sometimes we have `**kwargs` and sometimes we have only `kwargs`. though I will have to test this.

I am not quite sure what you mean by:
> You could move the arguments that are only passed to self.get to kwargs here as well. Also, are there plugins that make use of this function? 

I think your point about defaults is well made - I will make this change as soon as I get time to test it.